dask/dataframe/tests/test_categorical.py

```diff
@@ -193,16 +192,16 @@ def test_unknown_categoricals(shuffle_method, numeric_only):
     assert_eq(result, expected)
 
 
-def test_is_categorical_dtype():
+def test_isinstance_categorical_dtype():
```

Comment:
IIUC I think this test can actually be removed. Even prior to this PR, it looks like it was just testing `pandas` `is_categorical_dtype` method. This was added a while ago in https://github.com/dask/dask/pull/1602 where we had some extra logic around where `is_categorical_dtype` needed to be imported from `pandas`. Since we don't have that anymore, we should be good to remove this test